tsconfig.json

```diff
@@ -1,33 +1,24 @@
 {
-  "compilerOptions": {
-    "composite": true,
-    "emitDeclarationOnly": false,
-    "declaration": true,
-    "declarationMap": true,
-    "strictPropertyInitialization": false,
-    "strictNullChecks": false,
-    "module": "amd",
-    "target": "ES2017",
-    "moduleResolution": "node",
-    "isolatedModules": true,
-    "esModuleInterop": true,
-    "sourceMap": true,
-    "outDir": "./build",
-    "typeRoots": [
-      "./node_modules/@types",
-      "./node_modules/matrix-type-declarations"
-    ],
-    "lib": [
-      "es2016",
-      "dom"
-  ],
-    "jsx": "react-jsx"
-  },
-  "paths": {
-  },
-
-  "include": [
-     "src/**/*.ts",
-     "src/**/*.tsx"
-  ]
+    "compilerOptions": {
+        "jsx": "react-jsx",
+        "rootDir": ".",
+        "sourceMap": true,
+        "declaration": false,
+        "moduleResolution": "node",
+        "emitDecoratorMetadata": true,
+        "experimentalDecorators": true,
+        "target": "ESNext",
+        "module": "ESNext",
+        "lib": ["ESNext", "dom"],
+        "skipLibCheck": true,
+        "skipDefaultLibCheck": true,
+        "baseUrl": ".",
+        "allowJs": false,
+        "esModuleInterop": false,
+        "allowSyntheticDefaultImports": true,
+        "strict": false,
```

Comment:
wondering, how hard it would be to make the boilerplate `strict: true` compatible. 
at other hand, it's probably ok to treat plugins in a bit of a relaxed way.

Reply:
Oh that would be great! :) We've bitten off pieces of strict compilation in some places. I think the whole PRINT module is that way. I think of strict as state of the art, and would rather ratchet up to it wherever possible...

Reply:
I tried to enable it and it's a lot of ts errors inside the class components, I would definitely aim to `strict: true` otherwise it's just useless to have ts on the project at all